flink-runtime/src/test/java/org/apache/flink/runtime/state/ttl/MockTtlTimeProvider.java

```diff
@@ -26,4 +26,8 @@
 	public long currentTimestamp() {
 		return time;
 	}
+
+	public void setCurrentTimeStamp(long timeStamp) {
```

Comment:
```suggestion
	public void setCurrentTimestamp(long timestamp) {
```

A lowercase for `S`?